tests/EvmAcceptanceTests/test/ContractRevert.js

```diff
@@ -0,0 +1,29 @@
+const { expect } = require("chai");
+const { ethers, web3 } = require("hardhat");
+const general_helper = require("../helper/GeneralHelper");
+const web3_helper = require("../helper/Web3Helper");
+const zilliqa_helper = require("../helper/ZilliqaHelper");
+assert = require('chai').assert;
+
+
+describe("Revert Contract Deployment", function () {
+  let contract;
+  before(async function () {
+    contract = await web3_helper.deploy("Revert");
+  });
+
+
+  it("Will revert the contract", async function () {
```

Comment:
can you please be more verbose? `Will revert the contract when something is done`